src/platform/web/ui/general/html.ts

```diff
@@ -104,8 +104,9 @@ export const TAG_NAMES = {
         "br", "a", "ol", "ul", "li", "div", "h1", "h2", "h3", "h4", "h5", "h6",
         "p", "strong", "em", "span", "img", "section", "main", "article", "aside", "del", "blockquote",
         "table", "thead", "tbody", "tr", "th", "td", "hr",
-        "pre", "code", "button", "time", "input", "textarea", "label", "form", "progress", "output", "video"],
-    [SVG_NS]: ["svg", "circle"]
+        "pre", "code", "button", "time", "input", "textarea", "select", "option", "label", "form",
+        "progress", "output", "video"],
```

Comment:
The diff isn't obvious here but I just added `"select", "option"` to this list and put the others on a new line.